### ISSUE TITLE
Metrics in HB multi outputs job don't get correctly

### ISSUE THREAD

**Issue by taoyun951753** [alibaba/HybridBackend#90]
# Current behavior
When running multi outputs job with HB, Log doesn't get  the metrics of second output 
![B539969A-1C11-49AB-AE6D-1EC016B58F9C](https://user-images.githubusercontent.com/30410832/202602488-a11ad9c2-326a-4dcf-8d23-9d489a2cef75.png)

As show above, the log gets the output_1_auc metric but leave out the  output_2_mean_squared_error metric and   output_2_mean_absoluted_error metric

# Expected behavior
I want  all the metrics can get correctly

# System information
- GPU model and memory:
- OS Platform:
- Docker version:
- GCC/CUDA/cuDNN version:
- Python/conda version:
- TensorFlow/PyTorch version:

# Code to reproduce

```python
```

# Willing to contribute

Yes
